pkg/cliplugins/bind/plugin/bind_compute.go

```diff
@@ -235,22 +236,43 @@ func (o *BindComputeOptions) applyAPIBinding(ctx context.Context, client kcpclie
 	}
 
 	existingAPIExports := sets.NewString()
+	var bindingsToPatch []*apisv1alpha1.APIBinding
 	for _, binding := range apiBindings.Items {
 		if binding.Spec.Reference.Workspace == nil {
 			continue
 		}
-		existingAPIExports.Insert(fmt.Sprintf("%s:%s", binding.Spec.Reference.Workspace.Path, binding.Spec.Reference.Workspace.ExportName))
+		apiExportKey := fmt.Sprintf("%s:%s", binding.Spec.Reference.Workspace.Path, binding.Spec.Reference.Workspace.ExportName)
+		existingAPIExports.Insert(apiExportKey)
+		if desiredAPIExports.Has(apiExportKey) {
+			if len(binding.Labels) == 0 || len(binding.Labels[apisv1alpha1.LabelAPIBindingKindKey]) == 0 {
```

Comment:
Should we check when the label is already there with another value, and fail (to be prepared for the time other values my come) ?

Reply:
Thanks !